README.md

```diff
@@ -1,6 +1,24 @@
+![Screenshot](https://github.com/EppMouse/Eventual/raw/master/screen.PNG)

```

Comment:
Don't link to an external site as it may become unavailable in the future. Point it to the screenshot that you add alongside your readme.